Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/eventregistration/attendee/confirm/ConfirmCheckInViewModel.kt

```diff
@@ -28,11 +29,14 @@ class ConfirmCheckInViewModel @AssistedInject constructor(
 ) : CWAViewModel() {
     private val traceLocation = MutableStateFlow(verifiedTraceLocation.traceLocation)
     private val createJournalEntry = MutableStateFlow(true)
-    private val checkInLength = MutableStateFlow(
-        Duration.standardMinutes(
-            verifiedTraceLocation.traceLocation.defaultCheckInLengthInMinutes?.toLong() ?: 0L
-        )
+
+    private val truncatedDefaultCheckInLength = roundToNearestValidDuration(
+        verifiedTraceLocation.traceLocation.defaultCheckInLengthInMinutes ?: 0
     )
+    private val checkInLength = MutableStateFlow(truncatedDefaultCheckInLength)
+
+    private fun roundToNearestValidDuration(minutes: Int): Duration =
+        Duration.standardMinutes((minutes.toFloat() / 15).roundToLong() * 15)
```

Comment:
Could you please Extract `15` into a constant and give it a descriptive name

Reply:
Will adapt this in my upcoming PR!